Tests that fail when removing brackets:

```
FAILED tests/io/pyaro/test_read_pyaro.py::test_pyarotoungriddeddata_reading
FAILED tests/io/pyaro/test_read_pyaro.py::test_pyarotoungriddeddata_reading_kwargs
FAILED tests/io/pyaro/test_read_pyaro.py::test_pyarotoungriddeddata_reading_extra_metadata
FAILED tests/io/pyaro/test_read_pyaro.py::test_pyarotoungriddeddata_stations
```

```
TypeError: 'method' object is not subscriptable
TypeError: object of type 'method' has no len()
```

I suspect pyaro is returning a method instead of calling the method somewhere which this was supposed to mask?

I think this is the source of the error: https://github.com/metno/pyaro/blob/87ac570e5f657b4df0f92017c5fac32d6c873b2d/src/pyaro/timeseries/Wrappers.py#L49

--- a/pyaerocom_env.yml
+++ b/pyaerocom_env.yml
@@ -31,7 +31,7 @@ dependencies:
   - pip:
     - geojsoncontour
     - geocoder_reverse_natural_earth >= 0.0.2
-    - pyaro >= 0.0.10
+    - pyaro >= 0.0.12
     - aerovaldb >= 0.1.1
 ## testing
   - pytest >=7.4

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -47,7 +47,7 @@ dependencies = [
     'cf-units>=3.1',
     "pydantic>=2.7.1",
     "pyproj>=3.0.0",
-    "pyaro>=0.0.10",
+    "pyaro>=0.0.12",
     "pooch>=1.7.0",
     "psutil>=5.0.0",
 ]

